app/src/androidTest/java/org/mozilla/focus/activity/screenshots/SettingsScreenshots.java

```diff
@@ -99,9 +118,68 @@ public void takeScreenShotsOfSettings() throws Exception {
         TestHelper.settingsHeading.waitUntilGone(waitingTime);
         Screengrab.screenshot("SearchEngine_Add_Search_Engine");
         TestHelper.pressBackKey();
+        TestHelper.pressBackKey();
 
-        /* scroll down */
+/* Tap autocomplete menu */
+        final String urlAutocompletemenu = getString(R.string.preference_subitem_autocomplete);
+        onData(withTitleText(urlAutocompletemenu))
+                .perform(click());
+        onView(withText(getString(R.string.preference_autocomplete_custom_summary)))
+                .check(matches(isDisplayed()));
+        Screengrab.screenshot("Autocomplete_Menu_Item");
+
+        /* Enable Autocomplete, and enter Custom URLs */
+        onView(withText(getString(R.string.preference_autocomplete_custom_summary)))
+                .perform(click());
+        /* Add custom URL */
+        onView(withText(getString(R.string.preference_autocomplete_subitem_customlist)))
+                .perform(click());
+        device.waitForIdle();
+        final String addCustomURLAction = getString(R.string.preference_autocomplete_action_add);
+        onView(withText(addCustomURLAction))
+                .check(matches(isDisplayed()));
+        Screengrab.screenshot("Autocomplete_Custom_URL_List");
+
+        onView(withText(addCustomURLAction))
+                .perform(click());
+        onView(withId(R.id.domainView))
+                .check(matches(isDisplayed()));
+        Screengrab.screenshot("Autocomplete_Add_Custom_URL_Dialog");
+        onView(withId(R.id.domainView))
+                .perform(typeText("screenshot.com"), closeSoftKeyboard());
+        onView(withId(R.id.save))
+                .perform(click());
+        device.waitForIdle();
+        onView(withText(addCustomURLAction))
+                .check(matches(isDisplayed()));
+
+        /* Remove menu */
+        final String removeMenu = getString(R.string.preference_autocomplete_menu_remove);
+        openActionBarOverflowOrOptionsMenu(InstrumentationRegistry.getContext());
+        device.waitForIdle();
+        onView(withText(removeMenu))
+                .check(matches(isDisplayed()));
+        Screengrab.screenshot("Autocomplete_Custom_URL_Remove_Menu_Item");
+        onView(withText(removeMenu))
+                .perform(click());
+        device.waitForIdle();
```

Comment:
When using Espresso you shouldn't need to call waitForIdle(). Espresso will always wait for the device to idle before doing/checking anything.